packages/larva-patterns/modules/button/button.wide.js

```diff
@@ -0,0 +1,8 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const button = clonedeep( require( './button.prototype' ) );
+
+button.button_classes = ' lrv-u-border-radius-100 lrv-u-text-align-center lrv-u-padding-a-075 lrv-u-display-inline-block lrv-u-width-300 lrv-u-max-width-100p';
```

Comment:
Several changes needed here:

1. No leading space, since the entire string is overwritten
2. No widths should be set here because the block provides controls for those
3. The border-radius, text-align, and display classes aren't needed as those are set by the class that renders the block.